src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -23,6 +23,13 @@ public class DurableTaskOptions
         private string resolvedHubName;
         private string defaultHubName;
 
+        /// <summary>
+        /// The DefaultVersion value provided here will be automatically assigned to every orchestration
+        /// instance created by this app, and this value will be available on IDurableOrchestrationContext
+        /// passed to each orchestrator function replay invocation of this orchestration instance.
+        /// </summary>
+        public string DefaultVersion { get; set; }
```

Comment:
In the portable SDK, we have versioning broken down into two different options, the client and the worker. 

Do we want to have this in a `Client` subsection to match that? It seems like in durable functions, all the options share the same options file.

Reply:
We don't have a client/worker split in Durable Functions, so I think it makes sense to leave this as-is.